Client/Frontend/Toolbar+URLBar/LockButton.swift

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class LockButton: UIButton {
+
+    // MARK: - Variables
+
+    var selectedTintColor: UIColor!
+    var unselectedTintColor: UIColor!
+    var disabledTintColor = UIColor.Photon.Grey50
```

Comment:
if there's an easy quick fix for this, let's do it because it's weird like this.